services/minespace-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -84,10 +95,77 @@ const confirmationSubmission = (props) =>
     </Col>
   );
 
-const renderInitialReport = (props) => (
+const incidentStatusCalloutContent = (statusCode) => {
+  switch (statusCode) {
+    case "AFT":
+      return {
+        message:
+          "You determined that this incident was a dangerous occurence please provide investigation documentation in the final documentation section.",
+        title: "This Incident Requires Final Investigation Documentation",
+        severity: Strings.CALLOUT_SEVERITY.warning,
+      };
+    case "FRS":
+      return {
+        message:
+          "This incident has not yet been reviewed by ministry staff. You will be notified if further clarification is required.",
+        title: "This Incident is Pending Ministry Review",
+        severity: Strings.CALLOUT_SEVERITY.warning,
+      };
+    case "INV":
+      return {
+        message:
+          "This incident has been reviewed and is currently under investigation. You will be notified if further clarification is required.",
+        title: "This Incident is under investigation",
+        severity: Strings.CALLOUT_SEVERITY.warning,
+      };
+    case "UNR":
+      return {
+        message:
+          "This incident has been reviewed and is currently under investigation. You will be notified if further clarification is required.",
+        title: "This Incident is under investigation",
+        severity: Strings.CALLOUT_SEVERITY.warning,
+      };
+    case "MIU":
+      return {
+        message:
+          "This incident has been reviewed and is currently under investigation. You will be notified if further clarification is required.",
+        title: "This Incident is under investigation",
+        severity: Strings.CALLOUT_SEVERITY.warning,
+      };
+    case "CLD":
+      return {
+        message: "This incident has been reviewed and has been closed.",
+        title: "This Incident Has been closed",
+        severity: Strings.CALLOUT_SEVERITY.warning,
+      };
+    default:
+      return null;
+  }
+};
+
+const renderIncidentStatusCallout = (props) => {
+  const { status_code } = props.incident;
+  const { title, message, severity } = incidentStatusCalloutContent(status_code);
+
+  return (
+    <Callout
+      message={
+        <div>
+          <h4 style={{ color: "#313132", fontWeight: 700 }}>{title}</h4>
```

Comment:
is there a reason there is a colour code inline here?

Reply:
This code was used to keep the styles similar with NoD which has this in it's own stylesheet. I didn't feel it was necessary to create a stylesheet for this component for one thing.